src/v/rpc/client_probe.h

```diff
@@ -57,7 +57,7 @@ class client_probe {
     }
 
     void read_dispatch_error(const std::exception_ptr& e) {
-        rpclog.error("Error dispatching client reads: {}", e);
+        rpclog.warn("Error dispatching client reads: {}", e);
```

Comment:
```
rpc - Error dispatching client reads: seastar::broken_promise (broken promise)
```

is this tracked somewhere? broken promise is generally always a bug.

Reply:
Good question. Seems like we need to differentiate the "benign" error and treat it as such, and fix the broken promise.

For issues, I just found this one [1587](https://github.com/vectorizedio/redpanda/issues/1587)

Reply:
i think this is ok. warn vs error is a minor distinction

Reply:
PR [2917](https://github.com/vectorizedio/redpanda/pull/2917) looks related. I think it fixes this particular error. I will remove this part of the PR.

Reply:
sorry i think my comments above didn't get submitted before you pushed